File: android/src/main/kotlin/com/syncrhrohealth/bandpass_ble_connection/core/command/DeviceNameGetCmd.kt
package com.syncrhrohealth.bandpass_ble_connection.core.command

import com.syncrhrohealth.bandpass_ble_connection.core.command.builder.CommandBuilder
import com.syncrhrohealth.bandpass_ble_connection.core.enum.CommandCode
import com.syncrhrohealth.bandpass_ble_connection.core.handler.DeviceHandler


object DeviceNameGetCmd {
    fun send(handler: DeviceHandler) {
        val builder = CommandBuilder.build(CommandCode.PACKET_CMD_DEVICE_NAME_GET)
        handler.write(builder)
    }
}

File: ios/Classes/Core/Command/DeviceNameGetCmd.swift
//
//  DeviceNameGetCmd.swift
//  Runner
//
//  Created by MAC on 11/3/25.
//

import Foundation

struct DeviceNameGetCmd {
    static func send(handler: DeviceHandler) {
        let builder = CommandBuilder.build(commandCode: .packetCmdDeviceNameGet)
        handler.write(data: builder)
    }
}
